src/EventCalendar.js

```diff
@@ -42,24 +66,111 @@ class EventCalendar {
      *    { date: 'YYYY-MM-DD' }
      * ];
      */
-    init (startDate, endDate, events) {
-        if (typeof this.$html === 'undefined' || !this.$html.length) {
+    init (options = {}) {
+        let _self = this;
+
+        if (typeof _self.$html === 'undefined' || !_self.$html.length) {
             throw new Error('$html must be passed to EventCalendar');
         }   
 
-        let $container = this.$html.find('.js-event-calendar');
+        let $container = _self.$html.find('.js-event-calendar');
 
         if (!$container.length) {
             throw new Error('EventCalendar requires a .js-event-calendar element present in the DOM');
         }
 
-        let _self = this,
-            clndrSelected = startDate ? moment(startDate) : moment(new Date()),
-            clndrStart = startDate ? moment(startDate) : moment(new Date()),
-            clndrEnd = endDate ? moment(endDate) : moment(new Date()).add(15, 'years'),
-            clndrEvents = events ? events : [],
-            $weekdayPicker = _self.$html.find('.js-ercal-weekdays'),
-            clndrTemplate = `
+         // Set the locale
+         _self.setLocale(options.locale);
+
+        /**
+         * It's possible to provide conflicting start dates via startDate init option and startDateField.val, 
+         * they will be used in this priority order:
+         * 
+         *  1. startDateField.val (visible to the user in the UI so takes priority)
+         *  2. startDate
+         *  3. today
+         */
+        if (typeof options.startDateField !== 'undefined' && options.startDateField.length) {
+            _self.$startDateField = _self.$html.find(options.startDateField);
+
+            // If startDate is undefined, use 'today' for the following startDate calculation
+            let startDate = (typeof options.startDate !== 'undefined') ? options.startDate : _self.today.format(_self.localeFormat);
```

Comment:
This is perhaps a preference, but those brackets around your ternary expression are not required